rom-hacking/scripting/animation-commands/ac-down.mdx

```diff
@@ -0,0 +1,42 @@
+# (769) _AC_DOWN
+
+## Effect
+
+Moves the entity of the animation down over a given amount of frames per tile.
+
+If the **frames** argument is negative, 16 is used, or 8 if the players DashFlag is set.
```

Comment:
DashFlag is a unique term, create a file and link to it where referenced. It can be a stubbed page in the meantime.

Reply:
@SaltContainer do we want to keep this mention in there or just remove it entirely?